volume-providers/vpc/vpcclient/vpcvolume/get_volume.go

```diff
@@ -55,7 +55,7 @@ func (vs *VolumeService) GetVolumeByName(volumeName string, ctxLogger *zap.Logge
 
 	// Get the volume details for a single volume, ListVolumeFilters will return only 1 volume in list
 	filters := &models.ListVolumeFilters{VolumeName: volumeName}
-	volumes, err := vs.ListVolumes(1, filters, ctxLogger)
+	volumes, err := vs.ListVolumes(1, "", filters, ctxLogger)
```

Comment:
why `1` and "" are hard coded?

Reply:
`GetVolumeByName` methods uses `ListVolumes` calls to get volume details. Here `ListVolumes` method filters volume based on name and return volume detail. 
and here we want to filter 1 volume by name so no need to put `start` volID... that way `ListVolumes` method will start searching the partivular volume by name from beginning.